komga-webui/src/locales/en.json

```diff
@@ -51,24 +51,24 @@
     "button_scan": "Scan",
     "button_select_series": "Select Series",
     "field_import_path": "Import from folder",
-    "info_part1": "This screen lets you import files that are outside your existing libraries. You can only import files into existing Series, in which case Komga will move or copy the files into the directory of the chosen Series.",
```

Comment:
Much shorter. The last two sentences are redundant and counterproductive.

Reply:
The much shorter sentence leaves out the fact that you can only import files from outside your libraries. This is a very important point that needs to be stated.

As for the repetition, it is important, because those concepts are not trivial, and some people asked about it on the support channels.

Reply:
I included that after your comment.